PokeDemo/Models/PokemonResult.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Pokemon.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 05/07/22.
+//
+
+import Foundation
+
+struct PokemonResult: Decodable {
+    var count: Int?
```

Comment:
Remove optionals

Reply:
Covered in 4bf40a3f8029568b019914197f8773b6da2e3d75. Previous is kept optional as it is coming null